### ISSUE TITLE
not working in node REPL

### ISSUE THREAD

**Issue by roccomuso** [ebceu4/waves-crypto#7]
Please note that this will be true https://github.com/wavesplatform/waves-crypto/blob/master/src/index.ts#L223 in node.js REPL.
Thus will not work, it will try to call a browser method (`.getRandomValues`) inside the node.js repl environment.

**Comment by siemarell** [comment]
I've checked. It works both in browser and in nodejs. Could you please elaborate? Maybe I'm missing something.

**Comment by roccomuso** [comment]
node REPL.
If you launch the interactive shell and then use the module from there, you'll see the exception thrown.

**Comment by siemarell** [comment]
Yes, I've reproduced it. Could you help me, how do i check if my environment is nodejs REPL?

**Comment by roccomuso** [comment]
Sure.
These are good replacements for Node.js/REPL and Browser checks:
```js
const isBrowser = typeof window !== 'undefined' && ({}).toString.call(window) === '[object Window]'

const isNode = typeof global !== "undefined" && ({}).toString.call(global) === '[object global]'
```

**Comment by siemarell** [comment]
Thanks
